dptb/nn/dftb/interp.py

```
@@ -0,0 +1,366 @@
+        self.num_points = int(num_points)
+        self.device = device if device is not None else torch.device("cpu")
+        if isinstance(self.device, str):
+            self.device = torch.device(self.device)
+        self.dtype = dtype
+
+    def _get_sigma_pair(self, sym_i: str, sym_j: str) -> torch.Tensor:
+        try:
+            si = float(self.sigma_rep[sym_i])
+            sj = float(self.sigma_rep[sym_j])
+        except KeyError as e:
+            raise KeyError(f" Element not found in sigma_rep.") from e
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)